CriticalMass/MapViewController.swift

```diff
@@ -113,14 +133,18 @@ class MapViewController: UIViewController, MKMapViewDelegate {
         gpsDisabledOverlayView.isHidden = LocationManager.accessPermission == .authorized
     }
 
-    public lazy var followMeButton: UserTrackingButton = {
-        let button = UserTrackingButton(mapView: mapView)
-        button.tintColor = .navigationOverlayForeground
-        return button
-    }()
-
     // MARK: Notifications
 
+    @objc private func themeDidChange() {
+        let theme = ThemeController().currentTheme
```

Comment:
Shouldn't we inject the ThemeController or maybe pass the current theme as the notifications object instead of recreating a ThemeController

Reply:
Makes sense to inject it via from the AppController I think 👍